spec/index.bs

```diff
@@ -4193,16 +4213,18 @@ Issue(https://github.com/gpuweb/gpuweb/issues/693): add a limit to the number of
 
     Return `true`, if and only if, all of the following conditions are satisfied:
 
-        1. |descriptor|.{{GPUVertexStateDescriptor/vertexBuffers}}.length is less than or equal to 8
+        1. |descriptor|.{{GPUVertexStateDescriptor/vertexBuffers}}.length is less than or equal to
+            {{GPULimits/maxVertexBuffers|GPULimits.maxVertexBuffers}}
         1. Each |vertexBuffer| layout descriptor in the list |descriptor|.{{GPUVertexStateDescriptor/vertexBuffers}}
             passes [$validating GPUVertexBufferLayoutDescriptor$](|vertexBuffer|, |vertexStage|)
+        1. For every |vertexBuffer| in |descriptor|.{{GPUVertexStateDescriptor/vertexBuffers}},
+            the sum of every |vertexBuffer|.{{GPUVertexBufferLayoutDescriptor/attributes}}.length combined
+            is less than or equal to {{GPULimits/maxVertexAttributes|GPULimits.maxVertexAttributes}}.
```

Comment:
Not sure if there's a better way to write this down but this is the best I've got.

Reply:
I'll tweak the grammar but this looks good.